tools/cert-sidecar/example/README.md

```diff
@@ -2,7 +2,7 @@
 
 This example shows how to use **cert-sidecar** with a simple https client-server with mutual authentication and deployment in docker-compose and kubernetes.
```

Comment:
Does we have an example for Kubernetes?

Reply:
No more, I removed this example...
I will remove that from readme

Reply:
:+1: